### PR TITLE
April fools day post

### PR DESCRIPTION
**Pull request by mkosmul**

### CODE REVIEW

File: _posts/2022-03-31-agile-team-natural-habitat.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "An Agile team in its natural habitat"
+author: [michal.kosmulski]
+tags: [tech, agile, testing, "April 1st"]
+---
+In this picture, we can see an Agile team in its natural habitat at Allegro:
+
```
Comment (gotomi) [inline _posts/2022-03-31-agile-team-natural-habitat.md:8]: It doesn't look good... 🤔 
Screenshot 2022-03-31 at 10 06 23

**Comment by mkosmul** [comment]
I got approval from Paweł Łagoda and Karolina Ziemba-Orłowska to publish this picture from inside the office since no screens or details of the office layout are visible.

File: _posts/2022-03-31-agile-team-natural-habitat.md (hunk 1 above)
Comment (mkosmul) [reply _posts/2022-03-31-agile-team-natural-habitat.md:9]: Good point - I added a custom excerpt

**Comment by mkosmul** [comment]
Piotr Kaźmierczak approved of the post via Slack (I sent him a screenshot of the rendered page)

**Review by gotomi — APPROVED** [review]

**Pull request merged**